### ISSUE TITLE
Missing label on search field

### ISSUE THREAD

**Issue by jgonyea** [getgrav/grav-plugin-simplesearch#148]
Hi,
The `simplesearch_searchbox.html.twig` is missing a label for the input field.  Using an accessibility checker (http://wave.webaim.org/) throws an error on the search field. 

Adding:
```
<input
            name="searchfield"
            class="search-input"
            aria-label="search"
```
will correct the error.
